### DISCUSSION TITLE
Modifying .env file after running docker-compose up -d

### DISCUSSION THREAD

**Discussion by HK770** [adonisjs/core#1514] [Help] [answered]
I am running MySQL and adonis (v 4.1.0) in docker. I want the user to be able to supply their mail trap credentials in my .env file after running docker-compose up -d and want the changes to b reflected without having to restart my container.  Please suggest how I can do this.
Doing MAIL_USERNAME=process.env.mailusername isn't working out, where mailusername is an environment variable

**Comment by McSneaky** [comment, marked as answer]
Yea.. You can't really change them after NodeJS app starts up. But you can change them in app with `process.env.MAIL_USERNAME = 'hurrdurr'`. It might have some really odd side efects tho. It's usually not too good idea to change environment variables on the runtime
